tests/test.tracking.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+var tracking = require('tracking');
+
+
+describe('tracking uninitialized', function() {
+
+  beforeEach(function() {
+    this.oldTrackingEnabled = tracking.enabled;
+    this.oldTrackingInitialized = tracking.initialized;
+    tracking.enabled = true;
+    tracking.initialized = false;
```

Comment:
I know it's not as sexy but this would probably be simpler as a module object since that's how you're using it:
````
var initialized = false;

module.exports = {
  init: () => {
    // ...
    initialized = true;
  },

  sendEvent: opts => {
    // ...
  },
};
````
I still think it would be easier to test in isolation by making a new instance of the class for each test though. This way you don't have to tear down or set up anything.